doc/classes/ParallaxBackground.xml

```diff
@@ -4,7 +4,7 @@
 		A node used to create a parallax scrolling background.
 	</brief_description>
 	<description>
-		A ParallaxBackground will use one or more [ParallaxLayer] nodes to create a parallax scrolling background. Each [ParallaxLayer] can be set to move at different speeds relative to the camera movement, this can be used to create an illusion of depth in a 2D game.
+		A [code]ParallaxBackground[/code] use one or more [ParallaxLayer] children nodes to create the parallax effect. Each [ParallaxLayer] can be set to move at different speeds relative to the camera movement or the [member scroll_offset] value, this can be used to create an illusion of depth in a 2D game. Commonly used with a [Camera2D]. If not, you must calculate and set manually the [member scroll_offset].
```

Comment:
- You don't need [code] tags around the class title
- use -> uses
- "children nodes" -> "child nodes" or "children"
- The original "a parallax effect" was grammatically correct

Several issues with this part:

> Each [ParallaxLayer] can be set to move at different speeds relative to the camera movement or the [member scroll_offset] value, this can be used to create an illusion of depth in a 2D game. Commonly used with a [Camera2D]. If not, you must calculate and set manually the [member scroll_offset].

Try this:

> Each [ParallaxLayer] can move at a different speed using the [member scroll_offset] value. . This creates an illusion of depth in a 2D game. If not used with a [Camera2D], you must manually calculate the [member scroll_offset].

Reply:
@cbscribe for the last, I guess you misunderstood what I meant with "camera movement or the [member scroll_offset]". 
 [member scroll_offset] isn't for set a different speed for each ParallaxLayer, it is the current position of the scrolling background, managed manually or automatically if a Camera2D is used, I will try to propose a clearer sentence.